src/SDKs/LabServices/Management.LabServices/Microsoft.Azure.Management.LabServices.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<Version>3.0.0</Version>
```

Comment:
Any reason this is 3.0.0 as your first SDK?

Reply:
For consistency with our other product, DevTestLabs